tests/cypress/support/commands.js

```diff
@@ -91,6 +91,75 @@ Cypress.Commands.add('headlessDeleteUser', (userId) => {
     cy.wait('@deleteUser');
 });
 
+Cypress.Commands.add('headlessDeleteUserByUsername', (username) => {
+    cy.headlessGetUserId(username).then((id) => {
+        cy.headlessDeleteUser(id);
+    });
+});
+
+Cypress.Commands.add('headlessGetSelfId', () => cy.window()
+    .its('cvat').should('not.be.undefined')
+    .then(async (cvat) => {
+        const { data: { id } } = await cvat.server.request('/api/users/self', { method: 'GET' });
+        return id;
+    }),
+);
+
+Cypress.Commands.add('headlessGetUserId', (username) => cy.window().its('cvat')
+    .should('not.be.undefined')
+    .then(async (cvat) => {
+        const { data: { results: [{ id }] } } = await cvat.server.request(
+            `/api/users?filter={"==":[{"var":"username"}, "${username}"]}`,
+            { method: 'GET' },
+        );
+        return id;
+    }));
+
+Cypress.Commands.add('changeUserActiveStatus', (authKey, accountsToChangeActiveStatus, isActive) => {
+    cy.request({
+        url: '/api/users?page_size=all',
```

Comment:
I’m not convinced that `page_size=all` is a good idea. In general, it’s a bad practice because it can lead to very heavy requests.

Do we actually need to fetch **all** users just to check an array of `accounts`? I think we should look for a smarter approach.
The `GET /users` API already supports `filter` and `search` parameters. We can try using those. I’m not sure `search` will accept multiple usernames, but we can always build a JSON `filter` expression to query exactly what we need.

This keeps the request light and avoids unnecessary load.